packages/app/cypress/e2e/runner/reporter.hooks.cy.ts

```diff
@@ -0,0 +1,123 @@
+import { loadSpec } from './support/spec-loader'
+import { reporterSnapshot } from './support/reporter-snapshot'
+
+describe('hooks', {
+  // Limiting tests kept in memory due to large memory cost
+  // of nested spec snapshots
+  numTestsKeptInMemory: 1,
+}, () => {
+  afterEach(function () {
+    // @ts-ignore
+    if (this.currentTest.state === 'passed') {
```

Comment:
i believe you can use `Cypress.currentTest` here

Reply:
`Cypress.currentTest` [does not expose](https://github.com/cypress-io/cypress/blob/develop/packages/driver/src/cypress.ts#L662-L665) the whole runnable, which seems intentional based on the [original PR](https://github.com/cypress-io/cypress/pull/16652#issuecomment-887622782). I can see the state being useful info to have through this API, worth logging a follow-up enhancement?

Reply:
You can also use `cy.state('test')` instead